EvidenceApi/V1/Gateways/NotifyGateway.cs

```diff
@@ -170,5 +172,22 @@ private DocumentType GetDocumentType(EvidenceRequest evidenceRequest)
 
         private string MagicLinkFor(EvidenceRequest evidenceRequest) => $"{_options.EvidenceRequestClientUrl}resident/{evidenceRequest.Id}";
         private string ResidentPageLinkFor(EvidenceRequest evidenceRequest) => $"{_options.EvidenceRequestClientUrl}teams/{_documentTypeGateway.GetTeamIdByTeamName(evidenceRequest.Team)}/dashboard/residents/{evidenceRequest.ResidentId}";
+
+        private string FormatDocumentTypes(EvidenceRequest evidenceRequest)
```

Comment:
This method checks the documentTypes in the evidence request and compares them to the document type ids in the DocumentTypes.json (config). It then returns the human-readable title, so proof-of-id is returned as Proof of ID. This is because the id of the document type is not necessarily equal to the human-readable text e.g. proof-of-rtb is mapped to Proof of previous Right to Buy discount.

Reply:
👍